python/pyspark/worker.py

```diff
@@ -89,6 +90,26 @@ def verify_result_length(*a):
     return lambda *a: (verify_result_length(*a), arrow_return_type)
 
 
+def wrap_pandas_group_map_udf(f, return_type):
+    def wrapped(*series):
+        import pandas as pd
+
+        result = f(pd.concat(series, axis=1))
```

Comment:
Can we remove `keys=columns`? The original includes it as the 3rd argument.

Reply:
series itself has a name attribute. `pd.concat` will use the name attribute of series to be the column name in dataframe.

Reply:
Oh, I see. Thanks!